skimage/graph/tests/test_spath.py

```diff
@@ -1,9 +1,11 @@
 import numpy as np
+import pytest
 import skimage.graph.spath as spath
 
 from skimage._shared.testing import assert_equal, assert_array_equal
 
 
+@pytest.mark.thread_unsafe
 def test_basic():
```

Comment:
Clarification: Same here, why isn't this test thread-safe? I don't see a warning or global state here.

Reply:
There is an ongoing Cython/CPython issue that is causing sporadic segfaults: https://github.com/cython/cython/issues/6718, this is completely unrelated to the actual scikit-image code.